packages/bigtable/system-test/bigtable.js

```diff
@@ -510,7 +509,7 @@ describe('Bigtable', function() {
       it('should get rows', function(done) {
         TABLE.getRows(function(err, rows) {
           assert.ifError(err);
-          assert.strictEqual(rows.length, 4);
+          assert(rows.length >= ROWS.length);
```

Comment:
This PR will need reworking, but currently some tests insert rows, which affects the state outside of the test block. 4 came from "3 in the set up hook, and 1 from the last test". If this test was run in isolation, that last test wouldn't have inserted a row, so we would have 3 not 4. Testing for "at least as many in the set up hook" is an easier solution than creating a table for each test.